versions.props

```diff
@@ -23,15 +26,9 @@ org.mockito:* = 3.11.0
 org.mpierce.metrics.reservoir:* = 1.1.3
 org.openjdk.jmh:* = 1.32
 org.slf4j:* = 1.7.30
-com.squareup:javapoet = 1.13.0
-com.google.testing.compile:compile-testing = 0.19
 
-# dependency-upgrader:OFF
-
-# caffeine 3.0 requires JDK 11+ see https://github.com/ben-manes/caffeine/releases/tag/v3.0.0
-com.github.ben-manes.caffeine:caffeine = 2.9.1
 
+# dependency-upgrader:OFF
 # Explicitly maintaining back-compat with dropwizard metrics
-io.dropwizard.metrics:* = 3.2.5
```

Comment:
this pin was causing problems locally:

```
$ gw checkClassUniqueness --write-locks

> Task :tritium-metrics-jvm:checkClassUniqueness FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tritium-metrics-jvm:checkClassUniqueness'.
> Could not resolve all dependencies for configuration ':tritium-metrics-jvm:runtimeClasspath'.
   > Could not resolve io.dropwizard.metrics:metrics-jvm:3.2.5.
     Required by:
         project :tritium-metrics-jvm
      > Cannot find a version of 'io.dropwizard.metrics:metrics-jvm' that satisfies the version constraints:
           Dependency path 'com.palantir.tritium:tritium-metrics-jvm:0.19.0-2-g8e11a17.dirty' --> 'io.dropwizard.metrics:metrics-jvm:3.2.5'
           Constraint path 'com.palantir.tritium:tritium-metrics-jvm:0.19.0-2-g8e11a17.dirty' --> 'com.palantir.tritium:tritium:0.19.0-2-g8e11a17.dirty' --> 'io.dropwizard.metrics:metrics-jvm:{strictly 3.2.6}' because of the following reason: Locked by versions.lock
           Constraint path 'com.palantir.tritium:tritium-metrics-jvm:0.19.0-2-g8e11a17.dirty' --> 'io.dropwizard.metrics:metrics-core:3.2.5' --> 'io.dropwizard.metrics:_:3.2.5' --> 'io.dropwizard.metrics:metrics-jvm:3.2.5' because of the following reason: belongs to platform io.dropwizard.metrics:_:3.2.5

   > Could not resolve io.dropwizard.metrics:metrics-jvm:{strictly 3.2.6}.
     Required by:
         project :tritium-metrics-jvm > project :
      > Cannot find a version of 'io.dropwizard.metrics:metrics-jvm' that satisfies the version constraints:
           Dependency path 'com.palantir.tritium:tritium-metrics-jvm:0.19.0-2-g8e11a17.dirty' --> 'io.dropwizard.metrics:metrics-jvm:3.2.5'
           Constraint path 'com.palantir.tritium:tritium-metrics-jvm:0.19.0-2-g8e11a17.dirty' --> 'com.palantir.tritium:tritium:0.19.0-2-g8e11a17.dirty' --> 'io.dropwizard.metrics:metrics-jvm:{strictly 3.2.6}' because of the following reason: Locked by versions.lock
           Constraint path 'com.palantir.tritium:tritium-metrics-jvm:0.19.0-2-g8e11a17.dirty' --> 'io.dropwizard.metrics:metrics-core:3.2.5' --> 'io.dropwizard.metrics:_:3.2.5' --> 'io.dropwizard.metrics:metrics-jvm:3.2.5' because of the following reason: belongs to platform io.dropwizard.metrics:_:3.2.5
```

Reply:
odd, however we use 3.2.6 internally, and we can likely bump to 4.x relatively soon.

Reply:
yeah, 3.2.6 is mostly due to Apache Spark dependency